### ISSUE TITLE
[influxdb] InfluxDB configs prior to the introduction of "autogen" cannot access historical data

### ISSUE THREAD

**Issue by rkoshak** [openhab/openhab#4840]
The relevant community thread is [here](https://community.openhab.org/t/influxdb-grafana-persistence-and-graphing/13761/143)

Unfortunately a separate thread wasn't created so there are some other conversations interspersed. The comments are primarily from, opus, theo, and fremel75.

## Expected Behavior

When calling persistence methods on objects to pull historic data from the DB in rules should return the historic value rather than generating an exception in the logs.

## Current Behavior

Prior to InfluxDB 1.x a retention policy (RP) named "default" was created and used for all the data stored. I also think the binding itself did not yet support creation of your own RP.

On version 1.x+ InfluxDB changed this behavior to use "autogen" rather than "default" as the default RP. Setting `retentionPolicy=default` works for saving data. However when calling a persistence method on an Item the following exception is raised:

CODE:
```
        val startOfDay = now.withTimeAtStartOfDay
        postUpdate(Weather_Temp_Max, Weather_Temperature.maximumSince(startOfDay, "influxdb").state)
        postUpdate(Weather_Temp_Min, Weather_Temperature.minimumSince(startOfDay, "influxdb").state)
```

ERROR:
```
2016-11-15 11:05:00.003 [DEBUG] [.internal.InfluxDBPersistenceService] - got a query
2016-11-15 11:05:00.003 [TRACE] [.internal.InfluxDBPersistenceService] - Filter: itemname: Weather_Temperature, ordering: ASCENDING, state: null,  operator: EQ, getBeginDate: Tue Nov 15 00:00:00 MST 2016, getEndDate: null, getPageSize: 2147483647, getPageNumber: 0
2016-11-15 11:05:00.003 [TRACE] [.internal.InfluxDBPersistenceService] - appending limit 2147483647
2016-11-15 11:05:00.004 [TRACE] [.internal.InfluxDBPersistenceService] - startEntryNum 0
2016-11-15 11:05:00.004 [DEBUG] [.internal.InfluxDBPersistenceService] - query string: select value from default.Weather_Temperature where  time > 1479193200s  limit 2147483647
2016-11-15 11:05:00.008 [ERROR] [ntime.internal.engine.ExecuteRuleJob] - Error during the execution of rule Update max and min temperatures: {"error":"error parsing query: found DEFAULT, expected identifier at line 1, char 19"}
```
Running the query directly through the REST API generates the same error.

## Possible Solution

Further experimentation reveals that if the `default` is placed in double quotes for the queries the calls succeed. See this post from [fremel75](https://community.openhab.org/t/influxdb-grafana-persistence-and-graphing/13761/174). However, setting `retentionPolicy="default"` results in no longer being able to save data and lots of errors in the log. 

Per [this post](https://community.openhab.org/t/influxdb-grafana-persistence-and-graphing/13761/177) it appears that if you put the RP name in quotes for any RP works so the solution may be as simple as adding quotes around the RP name for all queries.

## Steps to Reproduce (for bugs)

See above

## Context

I am unable to use InfluxDB stored historic data with Rules greatly reducing InfluxDB's utility.

## Your Environment

* Version used: openHAB 2 snapshot built 2.0.0.201611272028, InfluxDB Add-on 1.9.0.201611290210
* Environment name and version (e.g. Chrome 39, node.js 5.4, Java 8, ...): Java 1.8.0_111, InfluxDB 1.1 using a datastore originally created under 0.9
* Operating System and version (desktop or mobile, Windows 10, Raspbian Jessie, ...): Ubuntu 16.04.1 with InfluxDB running in Docker (official InfluxDB Docker image from dockerhub).

**Comment by gedejong** [comment]
Created a PR which fixes this issue: https://github.com/openhab/openhab/pull/4888 and possibly other issues wrt special characters in item names.

**Comment by 9037568** [comment]
For some reason, I can't assign this to @gedejong ...

**Comment by 9037568** [comment]
Closed via #4888
